### ISSUE TITLE
Brave always crash when opening a specific profile

### ISSUE THREAD

**Issue by nazbrok** [brave/brave-browser#7795]
## Description
I have 3 profiles on Brave. Since few days, when I open a specific profile it always crash all my brave instances. 
Last time this profil was working, it had few tab open when I closed it (one week or so) so now when it tries to open this profil it tries top load those website. 
The issue could be related to one of those website but unfortunatly since it crash right away I cannot see what sites it tries to load.

Maybe there is a possibility to open this profile without loading the websites ?

I uploaded 2 crash report :

```
Uploaded Crash Report ID 29d5a6ba5effeaf2 (Local Context: 015be2fd-2f3f-4384-bd54-6cc243a7e86b)
Crash report captured on Friday, January 17, 2020 at 9:55:25 AM, uploaded on Friday, January 17, 2020 at 9:56:49 AM

Uploaded Crash Report ID 453fdebf10174bb3 (Local Context: 3641bad9-4d92-4416-84ad-b93c04fc55bc)
Crash report captured on Thursday, January 16, 2020 at 5:39:14 PM, uploaded on Friday, January 17, 2020 at 9:57:23 AM

```

## Steps to Reproduce

   1. 
   2. 
   3. 

## Expected result:
doesn't crash

## Reproduces how often: 
every time I try to open this profile

## Brave version (brave://version info)
Version 1.5.40 Chromium: 79.0.3945.117 (Official Build) nightly (64-bit)
Windows 10

## Version/Channel Information:

- Can you reproduce this issue with the current release? Can't try
- Can you reproduce this issue with the beta channel? Can't try
- Can you reproduce this issue with the dev channel? Can't try
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? Cannot test as it crash right away
- Does the issue resolve itself when disabling Brave Rewards? Cannot test as it crash right away
- Is the issue reproducible on the latest version of Chrome?

**Comment by nazbrok** [comment]
Note : I cannot delete the profile either, when I click on "remove profil" , it show a window listing the browsing data and 1 second after it crash.

**Comment by nazbrok** [comment]
fixed with **Version 1.6.14 Chromium: 80.0.3987.87 (Official Build) nightly (64-bit)**
